tests/integrations/aws_lambda/test_aws.py

```diff
@@ -123,8 +148,6 @@ def test_handler(event, context):
         b'{"foo": "bar"}',
     )
 
-    assert response["FunctionError"] == "Unhandled"
```

Comment:
Why did this change?

Reply:
This test-case was failing when I ran the test-suite locally, apparently "FunctionError" parameter wasn't coming as part of response. Although I rechecked it now and it is working fine. I'll revert this line.